queries/make/highlights.scm

```diff
@@ -38,17 +38,35 @@
 
 (variable_assignment
  (word) @symbol)
+
+
+(define_directive
+ "define" @keyword
+ name: (word) @symbol
+ [
+  "="
+  ":="
+  "::="
+; ":::="
```

Comment:
What is `:::=`?

Reply:
[Immediately Expanded Variable Assignment Operator](https://www.gnu.org/software/make/manual/make.html#Immediate-Assignment). I had to comment it out because otherwise the query does not parse without erroring (the current `make` parser doesn't recognize it).